docs/source/installation.rst

```diff
@@ -2,38 +2,107 @@
 ---------------------------
 
+.. highlight:: bash
+
 If you use another debian derivate, chances are this still works for you.
 If your distribution uses a different package manager, simply replace the commandline in the first step (the package names might be slightly different). The links in :ref:`install_dependencies` contain additional information on how to get the required programs for your system.
 
 1. Install dependencies::
 
-      $ sudo apt-get install git python-pip python2.7-dev postgresql postgresql-server-dev-all postgresql-client
+   $ sudo apt-get install git python-pip python2.7-dev postgresql postgresql-server-dev-all postgresql-client
 
-2. Install AiiDA::
+2. Setup a virtualenv:
 
-      $ pip install -U setuptools pip
-      $ pip install virtualenv
+   .. code-block:: bash
+
+      $ pip install -U setuptools pip wheel virtualenv
       $ virtualenv ~/aiidapy # or equivalent
       $ source ~/aiidapy/bin/activate
-      (aiidapy) $ pip install -e git+https://github.com/aiidateam/aiida_core.git#egg=aiida_core --process-dependency-links --src=<folder/containing/aiida> 
-      (aiidapy) $ verdi quicksetup
+      (aiidapy) $ # the name of the active environment is indicated in front of the prompt
+
+3. Install aiida into the environment::
+   
+   (aiidapy) $ cd <where_you_want_the_aiida_sourcecode>
+   (aiidapy) $ git clone https://github.com/aiidateam/aiida_core
+   (aiidapy) $ pip install -e aiida_core[verdi_shell,ssh,REST] --process-dependency-links
+
+
+For the next step to work, postgres must be running on port ``5432``. This should automatically be the case once it is installed (on Ubuntu).
+If you have changed the default configuration of postgres, you will be asked for the details. Refer to the `PostgresQL`_ documentation for details on how to ensure your postgres is running. Read ``(aiidapy) $ verdi quicksetup --help`` to see how to supply information about your postgres configuration beforehand.
+
+4. Setup AiiDA::
+
+   (aiidapy) $ verdi quicksetup
 
 You will be asked for your user information. Be aware that this information will be associated with your experiments and results for sharing.
+Alternatively you can give your information as commandline options (use the ``--help`` option for a list of options).
+
+5. (optional) Add the verdi command to your PATH::
 
-3. Add the verdi command to your PATH::
+   $ # for bash:
+   $ echo "export PATH="${PATH}:~/aiidapy/bin/verdi" >> ~/.bashrc
 
-      $ # for bash:
-      $ echo "export PATH="${PATH}:~/aiidapy/bin/verdi" >> ~/.bashrc
+Instead you might create a shell alias or explicitly activate (``source ~/aiidapy/bin/activate``) the virtualenvironment before working with aiida.
+If you are using a virtualenv manager like `virtualenvwrapper`_, or `conda`_, it is possible to set up the environment so that activating it launches the aiida daemon and possibly initializes other things for you. How this is done is described in your virtualenv manager's documentation.
```

Comment:
That's actually critical. I think those tricks should somehow be explained because they are crucial to the everyday user, who will not like to have to go through the doc of virtualenvwrapper...
In particular, I still did not figure out how to cleanly get the verdi completion without adding an explicit evaluation of 'verdi completioncommand' inside my .bashrc (which is not clean if one wants to switch between different environments...)